gateway/src/lib.rs

```diff
@@ -559,6 +559,9 @@ pub mod tests {
 
             let docker_host = "/var/run/docker.sock".to_string();
 
+            let posthog_key = env::var("POSTHOG_KEY")
+                .unwrap_or_else(|_| "public.ecr.aws/shuttle-dev/deployer:latest".to_string());
```

Comment:
Perhaps we should just set this to "" for tests? We could also mock it for testing with wiremock, but this functionality is simple, so if adding mocking here will delay the release, it can wait.